src/app/(pages)/sicoesitems/page.js

```diff
@@ -0,0 +1,191 @@
+'use client'
+import React, { useState, useEffect } from "react";
+import Table from "@/components/table";
+import FormNationalTender from "@/components/Form/formNationalTender";
+import ModalCuce from "@/components/modal";
+import FormNewItem from "@/components/Form/newItem";
+import SicoesData from "@/components/sicoesData";
+import { usePostItemMutation, useGetItemsQuery } from "@/redux/services/itemsApi";
+import AddIcon from '@mui/icons-material/Add';
+import { transformData, newItemObject } from "@/app/functions/utilities";
+import { useDispatch, useSelector } from "react-redux";
+import { nextPage, prevPage, firsPage, lastPage, searchCuce } from "@/redux/slice/paginationSlice";
+
+const SicoesItems = () => {
+
+    const headers = [
+        {
+            accessorKey: 'cuce',
+            header: 'Cuce',
+        },
+        {
+            accessorKey: 'entity',
+            header: 'Entidad',
+        },
+        {
+            accessorKey: 'contractDescription',
+            header: 'Tipo Contratación',
+        },
+        {
+            accessorKey: 'modality',
+            header: 'Modalidad',
+        },
+        {
+            accessorKey: 'auction',
+            header: 'Subasta',
+            meta: {
+                filterVariant: 'select',
+            },
+        },
+        {
+            accessorKey: 'publishDateItem',
+            header: 'Fecha de Publicación',
+        },
+        {
+            accessorKey: 'presentationDate',
+            header: 'Fecha de Presentación',
+        },
+        {
+            accessorKey: 'stateAuction',
+            header: 'Estado',
+        },
+    ];
+
+
+    const sicoesDataFields = [
+        { label: 'CUCE', key: 'cuce' },
+        { label: 'Entidad', key: 'entity' },
+        { label: 'Tipo de Contrato', key: 'contract' },
+        { label: 'Modalidad', key: 'modality' },
+        { label: 'Descripción del Contrato', key: 'contractDescription' },
+        { label: 'Subasta', key: 'auction' },
+        { label: 'Estado de la Subasta', key: 'stateAuction' },
+        { label: 'Fecha de Presentación', key: 'presentationDate' },
+        { label: 'Fecha de Publicación', key: 'publishDateItem' },
+        { label: 'Archivos', key: 'Archivos' },
```

Comment:
please try to use only english: { label: 'Archivos', key: 'files' },{ label: 'Formularios', key: 'Forms }, etc